community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/LoadCsvAcceptanceTest.scala

```diff
@@ -381,6 +381,20 @@ class LoadCsvAcceptanceTest
     result.toList should equal(List(Map("field" -> "something")))
   }
 
+  test("should restrict file urls to be rooted within an authorized directory") {
+    val dir = createTempDirectory("loadcsvroot")
+
+    val db = new TestGraphDatabaseFactory()
```

Comment:
It is no big deal, especially if this is blocking something. The rest of the tests are not written this way they use `execute` and I merely pointed out that there is a mechanism in place to add custom configurations to the running database via overriding `createGraphDatabase`, as it is done here there will actually be two running databases which seems unnecessary. But I am not saying this must be done before merging if it is urgent.